packages/medusa/src/subscribers/order.js

```diff
@@ -43,7 +43,7 @@ class OrderSubscriber {
               region_id: order.region_id,
               order_id: order.id,
               value: i.unit_price,
-              balance: i.unit_price,
+              balance: i.subtotal / i.quantity,
```

Comment:
todo: value and balance should always be tax exclusive

Reply:
indeed, value should be subtotal as well. I suggest to store it in a const and then assign that value to both props. Also, as discussed, this part of the subscriber should be moved in the order createFromCart service method instead of being done asynchronously and be hidden. I suggest to create a new protected method which will take care of that and just call that method in the appropriate place in the order createFromCart